spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
👎  I also believe this is really just testing Rails associations, which is an issue beyond the responsibility of these specs and is besides tested elsewhere.

This test is also exceedingly expensive as written (You don't need to actually do db writes to test associations as you are doing)